app/views/pages/terms_conditions.html.erb

```diff
@@ -0,0 +1,181 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class='govuk-heading-xl'>
+      Terms and Conditions
+    </h1>
+
+    <h2 class='govuk-heading-l'>
+      All Users
+    </h2>
+
+    <h3 class='govuk-heading-m'>
+      Using the website
+    </h3>
+
+    <p class="govuk-body">
+      Please read these Terms of Use (“General Terms”) carefully before using this Teachers: claim back your student
+      loan repayments website (the “Service”).
+    </p>
+
+    <p class="govuk-body">
+      This Service is maintained for your personal use and viewing. Access and use by you of this Service constitutes
+      your acceptance of these General Terms. This takes effect from the date on which you first use this website. If
+      you do not agree to these terms, you must not use this website.
+    </p>
+
+    <p class="govuk-body">
+      If we change these General Terms we will post the revised document here with an updated effective date. If we
+      make significant changes to these terms, we may notify you by other means such as sending an email or posting a
+      notice on our home page.
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Information About Us
+    </h3>
+
+    <p class="govuk-body">
+      This Service is operated by [the Department for Education] (“<strong>we</strong>”, “<strong>our</strong>”, or
+      “<strong>us</strong>”). We are a central government department and have our registered office at
```

Comment:
Why are these `<strong>` when the other legal short terms aren't?

Reply:
Copied from sample page like that so I did't want to change it.